packages/suite/src/middlewares/wallet/discoveryMiddleware.ts

```diff
@@ -34,6 +35,19 @@ const discoveryMiddleware = (api: MiddlewareAPI<Dispatch, AppState>) => (next: D
         return action;
     }
 
+    // temporary workaround - do not show backup modal in function call getAddress() in exchanges
+    // see useCoinmarketExchangeForm (line 104)
+    const isCoinmarketExchange = prevState.router.route?.name === 'wallet-coinmarket-exchange';
+
+    if (
```

Comment:
add this condition to line 30:
if (action.type === UI.REQUEST_CONFIRMATION && (discoveryIsRunning || isCoinmarketExchange))